darts/utils/likelihood_models/classification.py

```diff
@@ -0,0 +1,159 @@
+from typing import Optional
+
+import numpy as np
+
+from darts.logging import get_logger, raise_log
+from darts.utils.likelihood_models.base import LikelihoodType
+from darts.utils.likelihood_models.sklearn import SKLearnLikelihood
+
+logger = get_logger(__name__)
+
+
+class ClassProbabilityLikelihood(SKLearnLikelihood):
+    def __init__(
+        self,
+        n_outputs: int,
+        random_state: Optional[int] = None,
+    ):
+        """
+        Classes probability likelihood.
+        This likelihood is used for classification tasks where the model predicts classes probability.
+
+        Parameters
+        ----------
+        n_outputs
+            The number of predicted outputs per model call. `1` if `multi_models=False`, otherwise
+            `output_chunk_length`.
+        random_state
+            Optionally, control the randomness of the sampling.
+        """
+        super().__init__(
+            likelihood_type=LikelihoodType.ClassProbability,
+            n_outputs=n_outputs,
+            random_state=random_state,
+            parameter_names=[],
+        )
+
+    def fit(self, model):
+        """
+        Fits the likelihood to the model.
+
+        Parameters
+        ----------
+        model
+            The model to fit the likelihood to. The model is expected to be fitted and have a `classes_` attribute.
+        """
+        if not hasattr(model, "classes_"):
+            raise_log(
+                ValueError(
+                    "The model must have a `classes_` attribute to fit the likelihood."
+                ),
+                logger,
+            )
+
+        classes = model.classes_
+        if type(classes) is not list:
+            classes = [classes]
+        self._classes = classes
+
+        unqiue_classes = {label for classes in classes for label in classes}
+        self._parameter_names = [
+            f"probability_class_{int(label)}" for label in unqiue_classes
+        ]
+        return self
+
+    def _estimator_predict(
+        self,
+        model,
+        x: np.ndarray,
+        **kwargs,
+    ) -> np.ndarray:
+        # TODO support multi-output with inequal number of classes
```

Comment:
Yes, this would be required before merging :) 

also adding comments about the dimensions would be very helpful (similar to the other likelihoods)